37_setting_up_production_like_kubernetes_cluster_part_3_13_dec_2022/scripts/batch-create-start.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
are you sure scripts should be here? do you have some changes in comparison with awesome-linux-config? And in any case I'm not convinced this is the right place to commit scripts (in contrast to .env.* files)